bar/90_contents/Engine/setAccessInfo.js

```diff
@@ -155,13 +155,54 @@ function(request) {
         var accessToken = setInfo.accessToken;
         var refreshToken = setInfo.refreshToken;
         var srcAccountName = setInfo.srcAccountName;
-        if (accessToken && refreshToken && srcAccountName){
+        if (!(accessToken && refreshToken && srcAccountName)){
```

Comment:
Please use simple logic.  The following does not evaluate all 3 values. The first one that fails will return exception.  

    if (!accessToken || !refreshToken || !srcAccountName)  {